drivers/timer/npcx_itim_timer.c

```diff
@@ -134,7 +134,7 @@ static int npcx_itim_start_evt_tmr_by_tick(int32_t ticks)
 	 * Get desired cycles of event timer from the requested ticks which
 	 * round up to next tick boundary.
 	 */
-	if (ticks <= 0) {
+	if (ticks <= 0 && ticks != K_TICKS_FOREVER) {
 		ticks = 1;
 	}
 
```

Comment:
it seems this could be expressed cleaner by moving to the if/else branches below:

```
if (ticks == K_TICKS_FOREVER) {
...
} else {
        if (ticks <= 0) {
                ticks = 1;
         }
...
}
```

Reply:
Indeed. It looks cleaner. I will modify it later.

Reply:
Done.